dockerfiles/sidecar/Dockerfile

```diff
@@ -19,8 +26,9 @@ ENV SUPERCRONIC_URL=https://github.com/aptible/supercronic/releases/download/v0.
     SUPERCRONIC=supercronic-linux-amd64 \
     SUPERCRONIC_SHA1SUM=5ddf8ea26b56d4a7ff6faecdd8966610d5cb9d85
 
-COPY cron/backup-cron-job /etc/crontabs/backup-cron-job
-COPY scripts scripts
+COPY --from=builder /workspace-data-sync/dockerfiles/sidecar/scripts/watcher ./watcher/watcher.go
```

Comment:
why are you copying the BINARY `scripts/watcher` (which is never referenced in this dockerfile) to a .go sourcefile `./watcher/watcher.go` ?